languages.toml

```diff
@@ -880,6 +880,31 @@ indent = { tab-width = 4, unit = "    " }
 name = "php"
 source = { git = "https://github.com/tree-sitter/tree-sitter-php", rev = "f860e598194f4a71747f91789bf536b393ad4a56" }
 
+[[language]]
+name = "php_only"
+scope = "source.php_only"
+injection-regex = "php_only"
+file-types = ["blade"]
+shebangs = ["php_only"]
+roots = ["composer.json", "index.php"]
+language-servers = [ ]
+indent = { tab-width = 4, unit = "    " }
+
+[[grammar]]
+name = "php_only"
+source = { git = "https://github.com/tree-sitter/tree-sitter-php", rev = "cf1f4a0f1c01c705c1d6cf992b104028d5df0b53", subpath = "php_only" }
+
+[[language]]
+file-types = ["blade", "blade.php"]
```

Comment:
See https://github.com/helix-editor/helix/blob/81ae768a4ea20543bc16246a46552a69204fd7c1/languages.toml#L2945-L2948 as an example. When we detect the extension we split only the last `.<ext>` so you need to use a `suffix` configuration as a workaround

Reply:
This is not enough, regular `php` still takes precedence because suffixes are checked last https://docs.helix-editor.com/languages.html#file-type-detection-and-the-file-types-key.

Reply:
Hmm, we might need to change `helix-core/src/syntax.rs` to check suffixes first since they're more specific. Though we also want to switch to globs and I believe that PR handles this correctly: https://github.com/helix-editor/helix/pull/8006. I'll give that PR another look when I get the chance

Reply:
I'll add a label that this PR is waiting on another PR. I would hold off on rebasing until that PR is merged so if we need to make any changes you don't have to deal with the conflicts

Reply:
Now that #8006 is merged, I rebased on master and everything should be working fine! :smiley: